README.md

```diff
@@ -55,7 +55,7 @@ Now we should be ready to run the server:
 Other things you can do:
 
 * Run the test suite with `npm test`
-* Check your code style with npm run jscs (according to the Google style guide)
+* Check your code style with `node_modules/eslint/bin/eslint.js census/<filename.js>` using the eslint config `/.eslintrc`.
```

Comment:
You can add a new `script` to `package.json` doing this on all the .js files in the project. For `eslint`, you can have an [ignore](http://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories) file to specify what should not be included in the linting process. Might save you a few typos or when you forget to lint something.

Also, if you add a `npm run lint` command as described above, you can [add it into the `posttest` section](http://blog.npmjs.org/post/127671403050/testing-and-deploying-with-ordered-npm-run-scripts) (see section *"Run scripts after testing"*), so linter runs automatically if all the tests are green. This way you won't be bothered with fixing linting issues until the code actually works.